R/peaks-functions.R

```diff
@@ -239,7 +239,7 @@ joinPeaks <- function(x, y, type = "outer", tolerance = 0, ppm = 10, ...) {
                         halfWindowSize = 2L, method = c("MAD", "SuperSmoother"),
                         snr = 0L, k = 0L, descending = FALSE, threshold = 0,
                         msLevel = spectrumMsLevel, ...) {
-    if (!(spectrumMsLevel %in% msLevel) || isTRUE(centroided))
+    if (!(spectrumMsLevel %in% msLevel))
```

Comment:
`joinPeaks` should work on spectra level as well I guess?

Reply:
haven't thought about that - so you mean to add a parameter `msLevel` to it?

Reply:
I was just asking because you removed the `isTRUE(centroided)` test here. `msLevel` is already present.

Reply:
I had to remove this test, because calling `pickPeaks` will now change `centroided` to `TRUE` immediately. So once we execute the processing queue would not get centroided anymore because `centroided` is already `TRUE`. This was the main problem why I introduced this change.